packages/apps/proof-of-us/src/utils/socket/store.ts

```diff
@@ -40,43 +40,15 @@ const ProofOfUsStore = () => {
       signees: [{ ...account, signerStatus: 'init', initiator: true }],
```

Comment:
removing all the extra data in proofs.
this was needed for having a leaderboard